src/utils/detect-server.js

```diff
@@ -100,7 +101,7 @@ module.exports.serverSettings = async (devConfig, flags, projectDir, log) => {
     settings.command = assignLoudly(devConfig.command.split(/\s/)[0], settings.command || null, tellUser('command')) // if settings.command is empty, its bc no settings matched
     settings.args = assignLoudly(devConfig.command.split(/\s/).slice(1), [], tellUser('command')) // if settings.command is empty, its bc no settings matched
   }
-  settings.dist = flags.dir || devConfig.publish || settings.dist // dont loudassign if they dont need it
```

Comment:
`devConfig.publish` already has `config.build.publish`. If we do replace `flags.dir` with `get(devConfig, 'build.publish')` then it will overwrite `config.dev.publish`

Reply:
Right! :+1: